color.h

```diff
@@ -2,18 +2,29 @@
 // Created by kirill on 20.03.2020.
 //
 
-#ifndef UNTITLED_COLOR_H
-#define UNTITLED_COLOR_H
+#ifndef SIGNATURE_COLOR_H
+#define SIGNATURE_COLOR_H
 
 #include <ostream>
 
-enum class color
+enum class Color
```

Comment:
items in enum classes should look like:
kSomeItemInEnumClass

Reply:
Done